storage/src/main/java/io/delta/storage/S3SingleDriverLogStore.java

```diff
@@ -223,8 +228,17 @@ private Iterator<FileStatus> listFromInternal(
             );
         }
 
+        FileStatus[] statuses;
+        if (fs instanceof LocalFileSystem || fs instanceof RawLocalFileSystem // This is needed for tests to pass
```

Comment:
nit: can you split `fs instanceof LocalFileSystem || fs instanceof RawLocalFileSystem` into 2 lines? that way the comment `This is needed for tests to pass` is more precise

Reply:
I believe the comment applies to both parts of the or statement. Should we still split it over two lines?

Reply:
oh, I see. can you say `These are needed for tests to pass` instead?

Reply:
Thanks!